src/mode/graphics.rs

```diff
@@ -85,8 +85,11 @@ where
         self.properties
             .set_draw_area((0, 0), (display_width, display_height))?;
 
+        // XXX Might need to change the draw area on 132x64?
+
         match display_size {
             DisplaySize::Display128x64 => self.properties.draw(&self.buffer),
+            DisplaySize::Display132x64 => self.properties.draw(&self.buffer[0..1056]),
```

Comment:
These numbers look like width*height/8, but I'm not sure if I'm right, or if some displays would need a different value. For 132x64 I don't know if we need to draw at an offset like [offset..offset+1056]

This wholeblock could be switched to this
```suggestion
self.properties.draw(&self.buffer[0..(display_width*display_height / 8)]);
```

Reply:
> These numbers look like width*height/8

Correct.

> For 132x64 I don't know if we need to draw at an offset

I don't think so; it's easier to understand if the buffer starts from zero.

> This wholeblock could be switched to this

It could, but I'd prefer to keep the explicit slice ranges. It's more obvious how many bytes are required by the display, and I want to change how this driver supports multiple resolutions soon anyway - a smaller display than the max 128x64 wastes an awful lot of RAM in the remaining part of the buffer. Tracked in #20.